Sources/Clickstream/Dependency/Clickstream/AutoRecord/AutoRecordEventClient.swift

```diff
@@ -35,10 +35,9 @@ class AutoRecordEventClient {
     }
 
     func onViewDidAppear(screenName: String, screenPath: String, screenHashValue: String) {
-        let screenUniqueId = getScreenUniqueId(screenHashValue)
-        if !isSameScreen(screenName, screenPath, screenUniqueId) {
+        if !isSameScreen(screenName, screenPath, screenHashValue) {
             recordUserEngagement()
-            recordScreenView(screenName, screenPath, screenUniqueId)
+            recordScreenView(screenName, screenPath, screenHashValue)
```

Comment:
Because it's a bug fix, let's have a test case to cover the changes.

Reply:
add the test case for cover the changes.